File: CS/Q338316.Web/CustomSessionUtilityService.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using DevExpress.XtraReports.Service.Native.Services;
using DevExpress.XtraPrinting;

namespace Q338316.Web {
    public class CustomSessionUtilityService : XRSessionUtilityService {
        protected override void ConfigurePrintingSystem(PrintingSystemBase ps) {
        }
    }
}

File: VB/Q338316.Web/CustomSessionUtilityService.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Web
Imports DevExpress.XtraReports.Service.Native.Services
Imports DevExpress.XtraPrinting

Namespace Q338316.Web
	Public Class CustomSessionUtilityService
		Inherits XRSessionUtilityService
		Protected Overrides Sub ConfigurePrintingSystem(ByVal ps As PrintingSystemBase)
		End Sub
	End Class
End Namespace